docs/changelog.rst

```diff
@@ -7,7 +7,7 @@ Changelog
 Develop
 -----------------
 * [BUGFIX] SuiteEditNotebookRenderer no longer break GCS and S3 data paths
-
+* [FEATURE] Add new exception: expect_multicolumn_sum_to_equal for `pandas` and `Spark` (thanks @chipmyersjr)!
```

Comment:
added change description to `Develop` section, along with thanks to @chipmyersjr